test/e2e/service_test.go

```diff
@@ -0,0 +1,115 @@
+// Copyright 2021 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package e2e
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	corev1 "k8s.io/api/core/v1"
+)
+
+// TestClusterIPHostAccess tests traffic from host to Cluster IP Service.
+func TestClusterIPHostAccess(t *testing.T) {
+	skipIfProviderIs(t, "kind", "TestClusterIPHostAccessOnKind tests on Kind.")
```

Comment:
I guess my confusion is why do we need to make Kind a special case. IIRC, `RunCommandOnNode` is available on Kind (it uses `docker exec` instead of SSH) and `curl` is installed on the Kind Nodes.

Reply:
I tried yesterday but it failed. 
https://github.com/antrea-io/antrea/runs/2761400464?check_suite_focus=true
```
 service_test.go:59: Error when running command on Node 'kind-control-plane', rc: 28, stdout: curl: (28) Failed to connect to 10.96.250.31 port 80: Connection timed out
        , stderr: , error: <nil>
```
On Jenkins, we need to update how the `ssh-config` is generated, so I guess something similar is needed for kind. I didn't have time yesterday to debug so I simply use hostnetwork client pod. I will try to solve the issue today.

Reply:
I don't know if you made progress on this. AFAIK, there should be no difference in accessing the Service using `RunCommandOnNode`, or from a hostNetwork Pod. So the error is surprising to me. Is it possible that it is a timing issue (e.g. kube-proxy rules not installed yet)?

Reply:
I am still trying to solve it. I tried ./test-e2e-kind.sh command on my local testbed and it works.
Thank you and that sounds possible. I will let it sleep a while and see it will work this time.

Reply:
@antoninbas The solution works. PTAL.